Sandbox/Program.cs

```diff
@@ -1,14 +1,79 @@
 ﻿using System;
+using System.IO;
 
 namespace Sandbox
 {
-    class Program 
+    class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World" + "\n" + "Press any key to continue.");
-            Console.ReadKey();
-              //nube
+            WriteFibonacci();
+            ReadFibonacci();
+
+        }
+        static void WriteFibonacci()
+        {
+            string path = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + "\\fibonacci.txt";
+            double next = 0;
+            double first = 0;
+            double second = 1;
+            using (StreamWriter sw = new StreamWriter(path))
+            {
+                    for (int y = 1; y < 101; y++)
```

Comment:
You can also do a y<=100;

Reply:
you can just do if((y%10) == 0)  no need for the extra variables or the ternary operator

Reply:
I agree, I do not know why I did it that way, it's simpler if I just combine and skip the ternary.

Let me know if you need me to do anything else with this one or if it can merge.  And if so merge, show help me with the next part...I created a new branch for writing to a prompted file name but I might've created the branch incorrectly.